hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/webapp/YarnJacksonJaxbJsonProvider.java

```diff
@@ -51,6 +53,9 @@ public YarnJacksonJaxbJsonProvider() {
   public ObjectMapper locateMapper(Class<?> type, MediaType mediaType) {
     ObjectMapper mapper = super.locateMapper(type, mediaType);
     configObjectMapper(mapper);
+    if (type == TimelineAbout.class) {
```

Comment:
@steveloughran I needed to add this to fix tests. I'm not sure what has changed in Jackson or other libs that requires this to be set here. Also, I would be concerned that other classes might be affected but that unit tests don't deserialize instances for those classes.